pkg/kubernetes/views/project-modify.html

```diff
@@ -55,6 +55,13 @@ <h3 class="modal-title" ng-if="project" translatable="yes">Change project</h3>
                 </td>
             </tr>
         </table>
+        <div class="alert dialog alert-info" ng-if="!project">
+          <span class="pficon pficon-info"></span>
+          <span translatable="yes">
+            A project is a namespace for images. It corresponds to a Docker repository.
```

Comment:
What I'm trying to show is how this value is used. It becomes a key part of how the images are addressed. How about something like:

"A project is a namespace for images. It is used as the repository name when referencing images, registry/**project**/image:[tag]."